interfaces/system_key.go

```diff
@@ -228,7 +235,7 @@ func WriteSystemKey() error {
 // to disk whenever apparmor-parser-mtime changes (in this manner
 // snap run only has to obtain the mtime of apparmor_parser and
 // doesn't have to invoke it)
-func SystemKeyMismatch() (bool, error) {
+func SystemKeyMismatch(promptingFlagEnabled bool) (bool, error) {
```

Comment:
feels like the inputs here should be a struct even if for now there is only one field, otherwise the bool here is fairly opaque